save.go

```diff
@@ -95,6 +96,22 @@ func dotPackage() (*Package, error) {
 	return p[0], nil
 }
 
+func projectPackages(dot *Package, a []*Package) []*Package {
```

Comment:
recent changes have done away with the dot package reference because we were only using the dir anyway.